config/metac.yaml

```diff
@@ -148,37 +148,22 @@ spec:
       resource: daemonsets
       updateStrategy:
         method: InPlace
-      labelSelector:
```

Comment:
You may want to avoid using `updateAny` in future if required. Metac now supports `ExplicitUpdates` & `ExplicitDeletes`.
You may want to track this via issue.

Reply:
Created an issue: https://github.com/mayadata-io/openebs-upgrade/issues/45